src/TeamListView.cpp

```diff
@@ -23,16 +23,16 @@ TeamListView::TeamListView(BRect frame, const char *name)
 	  B_FOLLOW_ALL_SIDES)
 {
 	SetInvocationMessage(new BMessage(TEAM_INV));
-	
+
 }
 
 void TeamListView::SelectionChanged()
 {
 	Window()->PostMessage(SELECTION_CHANGED);
 } */
 
-#include "TeamListView.h"
 #include "CLVColumn.h"
+#include "TeamListView.h"
```

Comment:
This reordering has broken the build. Now CLVColumn.h isn't aware of BView, for example. Revert this one.
Re-ordering of includes is best done in small steps with clean builds inbetween.
Do make sure to do a clean build before creating/updating a PR. The most innocuous little change has the potential to break things... :)

Reply:
My bad.  I'll let bach5000 make the fix and retest it.

Reply:
Oops.